libs/core/src/components/link/link.component.ts

```diff
@@ -55,6 +54,12 @@ export class GdsLink extends withMarginProps(
   @property()
   rel?: string
 
+  /**
+   * The label of the link.
```

Comment:
Add some more info here:

Use this to add an accessible label, for example when only an icon is used in the link, or if different copy is needed for screen reader users.